tox.ini

```diff
@@ -91,5 +91,5 @@ whitelist_externals =
 setenv =
     PYTHONPATH = {envtmpdir}/mongo-python-driver
 commands =
-    git clone --depth 1 https://github.com/mongodb/mongo-python-driver.git {envtmpdir}/mongo-python-driver
+    git clone --depth 1 https://github.com/mongodb/mongo-python-driver.git@v3.12 {envtmpdir}/mongo-python-driver
```

Comment:
I think this is `pip` syntax and might not work in a `git clone` command. For this you likely need to do
```
git clone --branch v3.12 https://github.com/mongodb/mongo-python-driver.git
```

Reply:
ahh ok updated!